docs/numbers.rst

```diff
@@ -86,6 +86,58 @@ the specification. The following table is just a relatively brief overview.
   +----------+-----------------------------------------------------------------+
 
 
+Rounding Modes
```

Comment:
+++++++

Thanks for the great docs!

Could you add a doctest or a note to the docstring for `babel.numbers.format_decimal`? Same for `format_currency` - especially around the interaction with custom currency formats and `currency_digits`. (omg so complicated)